### ISSUE TITLE
Bus on Login Page

### ISSUE THREAD

**Issue by alanzhu0** [tjcsl/ion#1457]
## Topic of discussion
- Discuss potentially adding bus map to be visible without logging in, similar to events 
- Consider also highlighting the user's bus even when logged out, using cookies (similar to dark mode) 
- Likely restrict to only while on FCPS network 
- Is this a security concern?

## Personal opinion (if applicable)
- I think if it's on FCPS WiFi it should probably be okay, but we'd probably have to check with admin
- It's not exactly a secret where each bus is, they're all labeled on their windows anyway...

**Comment by tonymac129** [comment]
Could I work on this issue? I'm not sure if it's possible to check if they're connected to FCPS Wifi though

**Comment by aarushtools** [comment]
> Could I work on this issue? I'm not sure if it's possible to check if they're connected to FCPS Wifi though

It is possible, but can you wait on this? We need to check with admin first

**Comment by tonymac129** [comment]
Sure, I can wait on this. In the meantime, are there any other open issues I could work on?

**Comment by aarushtools** [comment]
Yes, can you work on #1300 ? I think we might have had this issue recently while using it for tjStar (cc @shrysjain )

**Comment by TigerCodeHub** [comment]
Given admin doesn't want signage school maps public, they probably wouldn't want live bus placements online either - issue can probably be closed.
